pkg/apis/work/v1alpha2/well_known_labels.go

```diff
@@ -25,6 +25,9 @@ const (
 
 	// WorkNameLabel is added to objects to specify associated Work's name.
 	WorkNameLabel = "work.karmada.io/name"
+
+	// ObjectReferenceUIDLabel is added to objects to specify object's uid
+	ObjectReferenceUIDLabel = "objectference.k8s.io/uid"
```

Comment:
```golang
// Define annotations that are added to the resource template.
const (
	// ResourceTemplateUIDAnnotation is the annotation that is added to the manifest in the Work object.
	// The annotation is used to identify the resource template which the manifest is derived from.
	// The annotation can also be used to fire events when syncing Work to member clusters.
	// For more details about UID, please refer to:
	// https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids
	ResourceTemplateUIDAnnotation = "resourcetemplate.karmada.io/uid"
)
```
How about this annotation?

Reply:
That's great. I'm not very good at naming

Reply:
No worries, we need to know that the annotation/label is a kind of API, that's we have to take care of very carefully.